docs/pages/web/textfield.js

```diff
@@ -501,6 +501,78 @@ The first example shows an empty Textfield with \`maxLength\` set to 20 characte
 export async function getServerSideProps(): Promise<{
   props: { generatedDocGen: DocGen },
 }> {
+  const generatedDocGen = await docGen('TextField');
+
+  generatedDocGen.props.autoComplete = {
+    defaultValue: null,
+    required: false,
+    flowType: {
```

Comment:
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/4ec10fa5-c18c-4b23-9dbe-7cd54761d4ba)


I think you can just do that:

`  generatedDocGen.props.color.flowType.raw = '"bday" | "current-password" | "email" | "new-password" | "on" | "off" | "username"';
`

Reply:
Ravi suggested the raw type to have all types, so it's properly copied
https://pinterest.slack.com/archives/G01GN7CE99S/p1709073511530249?thread_ts=1709064809.271199&cid=G01GN7CE99S

Reply:
We don't copy the whole Flowtype from the Table, just:

$ElementType<React$ElementConfig<typeof TextField>, 'autoComplete'>

So Ravi's suggestion is not necessary unless I'm missing something

Reply:
Makes sense, will edit!